browser/ui/webui/brave_webui_source.cc

```diff
@@ -81,6 +81,11 @@ void CustomizeWebUIHTMLSource(const std::string &name, content::WebUIDataSource*
       }
     }, {
       std::string("rewards"), {
+        { "823882f2419bf6c16b8291e056ea4b52.svg", IDR_BRAVE_REWARDS_IMG_WALLET_ICON },
```

Comment:
it looks like these images are being included by webpack so I don't think we need this

Reply:
I guess I'm not clear on what is happening with the images in webpack so maybe we need them.

Reply:
yeah we need to add it otherwise they are not found

Reply:
it would be nice if webpack could bundle these somehow